trulens_eval/trulens_eval/tests/context_relevance_benchmark.ipynb

```diff
@@ -0,0 +1,297 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    from benchmark_frameworks.eval_as_recommendation import score_passages, compute_ndcg, compute_ece, precision_at_k
</pre><p>Can you add some info here on why these metrics are chosen and short explainers on how they are computed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/701/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>